core/src/main/java/com/google/errorprone/bugpatterns/formatstring/FormatString.java

```diff
@@ -33,17 +36,41 @@
 @BugPattern(name = "FormatString", summary = "Invalid printf-style format string", severity = ERROR)
 public class FormatString extends BugChecker implements MethodInvocationTreeMatcher {
 
+  private static final Matcher<ExpressionTree> FORMATTED_METHOD = instanceMethod().onExactClass("java.lang.String").named("formatted");
+
   @Override
   public Description matchMethodInvocation(MethodInvocationTree tree, final VisitorState state) {
-    ImmutableList<ExpressionTree> args = FormatStringUtils.formatMethodArguments(tree, state);
-    if (args.isEmpty()) {
+    final ImmutableList<ExpressionTree> args;
```

Comment:
nit: house style is to usually leave `final` off variables, can you leave these out?

(I realize there's a `final VisitorState state` right above this line...)